test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,31 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
Coding standard requires to include descriptive header comments for public methods